service-base/src/main/java/org/eclipse/hono/service/management/credentials/PasswordSecret.java

```diff
@@ -112,6 +112,9 @@ protected ToStringHelper toStringHelper() {
     @Override
     public void checkValidity() {
         super.checkValidity();
+        if (containsOnlySecretId()){
```

Comment:
Can we keep formatting consistent. There are a couple of places where there's no space after bracket and at some places there's a space in negation statement. We can also consider adding it to the check style.